### ISSUE TITLE
Error:(72) unknown element <receiver> found in the merged manifest

### ISSUE THREAD

**Issue by jayakrishnan1236** [dji-sdk/Mobile-SDK-Android#167]
```
    <uses-permission android:name="android.permission.USE_CREDENTIALS" />

    <receiver
        android:name="dji.logic.receiver.DJIPilotStartupReceiver"
        android:exported="true"
        android:permission="dji.permission.sdk.wifi" >
        <intent-filter>
            <action android:name="dji.pilot.STARTUP" />
            <action android:name="dji.go3.STARTUP" />
            <action android:name="dji.go4.STARTUP" />
        </intent-filter>
    </receiver>
<application...
```

The manifest merge is not happening correctly. The receiver is getting merged before the application tag.

Dji-Sdk version: 4.3.2.
Android Studio 3.0 beta 4

**Comment by jayakrishnan1236** [comment]
I dont have the same issue with DjI-SDK 4.2.1

**Comment by jayakrishnan1236** [comment]
The issue resolved by adding `android.enableAapt2=false` to `gradle.properties` file. More details here.
https://stackoverflow.com/questions/46560449/error72-unknown-element-receiver-found/46562310#46562310

**Comment by ZeroCool2u** [comment]
@jayakrishnan1236 This does not resolve the issue, this only hides the issue and degrades performance. As a temporary workaround this is acceptable, but there is a misplaced tag somewhere. [See this stack overflow thread](https://stackoverflow.com/questions/46948498/android-studio-3-0-manifest-error-unknown-element-action-found).

@oliverou Is there an ETA on when this will be fixed?

**Comment by GIS-HowardZhang** [comment]
@ZeroCool2u Couldn't agree more!  I faced the same issue and had to workaround it by disabling  the AAPT2 in `gradle.properties` file.
 After I have checked the [Manifest file structure](https://developer.android.com/guide/topics/manifest/manifest-intro.html#filestruct)  ,my app `AndroidManifest.xml` and the generated `AndroidManifest.xml` under **app-build-intermediates-manifests-full-debug**, I was more convinced that this issue was produced by DJI SDK after upgrading Android Studio to 3.0 and Android plugin to 3.0.0, which have  enabled aapt2 as default and changed some warnings to errors. Part of error structure in generated `AndroidManifest.xml`:

<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
          package="com.***.****"
          android:versionCode="1"
          android:versionName="1.0">
  .....

    <receiver
        android:name="dji.logic.receiver.DJIPilotStartupReceiver"
        android:exported="true"
        android:permission="dji.permission.sdk.wifi">
        <intent-filter>
            <action android:name="dji.pilot.STARTUP"/>
            <action android:name="dji.go3.STARTUP"/>
            <action android:name="dji.go4.STARTUP"/>
        </intent-filter>
    </receiver> 

    <application
......

Then `Error:(72) unknown element <receiver> found in the merged manifest` occurred. We can see the receiver tag is placed wrong although no receiver can be find in my manifest file at all. And it's all good before upgrading Android Plugin.

Hope dji resolve it soon!
